src/TextAnalysis.jl

```diff
@@ -74,6 +77,10 @@ module TextAnalysis
     end
     include(depsjl_path)
 
+    function __init__()
+        include(joinpath(@__DIR__, "Sequence Labelling/NER_DataDeps.jl"))
```

Comment:
Again, no `include` in `__init__`. Also this will conflict with #166 , so might fix  and merge that first.

Reply:
Done for now. Will resolve conflicts after #166 is in.